code/modules/spacepods/spacepod.dm

```diff
@@ -226,26 +229,36 @@
 	health = max(0, health - damage)
 	var/percentage = (health / initial(health)) * 100
 	occupant_sanity_check()
-	if(passengers && oldhealth > health && percentage <= 25 && percentage > 0)
+	if(passengers | pilot && oldhealth > health && percentage <= 25 && percentage > 0)
 		var/sound/S = sound('sound/effects/engine_alert2.ogg')
 		S.wait = 0 //No queue
 		S.channel = 0 //Any channel
 		S.volume = 50
-		for(var/mob/M in passengers)
-			M << S
-	if(passengers && oldhealth > health && !health)
+		if(pilot)
+			pilot << S
+		if(passengers)
+			for(var/mob/M in passengers)
+				M << S
+	if(passengers | pilot && oldhealth > health && !health)
 		var/sound/S = sound('sound/effects/engine_alert1.ogg')
 		S.wait = 0
 		S.channel = 0
 		S.volume = 50
-		for(var/mob/M in passengers)
-			M << S
+		if(pilot)
+			pilot << S
+		if(passengers)
+			for(var/mob/M in passengers)
```

Comment:
If you're doing this so often, perhaps you might want a proc to play a sound to all occupants of a pod, as well as a proc to display a message to all occupants, as well.